### PR TITLE
chore: Use of VAT Percent attribute from Firestore

### PR DESCRIPTION
**Pull request by jorelosorio**
Lukker https://github.com/AtB-AS/mittatb-app/issues/2475

This issue will refactor the values in configuration/other/vatPercent

### CODE REVIEW

**Review by monopoint — APPROVED** [review]
:+1:

File: src/configuration/FirestoreConfigurationContext.tsx [hunk 1]
```diff
@@ -37,6 +39,7 @@ const defaultConfigurationContextState: ConfigurationContextState = {
   userProfiles: [],
   modesWeSellTicketsFor: [],
   paymentTypes: [],
+  vatPercent: 0,
```
Comment (gorandalum) [inline src/configuration/FirestoreConfigurationContext.tsx:42]: The defaults (like `defaultVatPercent`) should/could probably also be used here? If so then also update the other variables.

File: src/reference-data/defaults.ts [hunk 2]
```diff
@@ -17,3 +17,4 @@ export const defaultModesWeSellTicketsFor: string[] = [
   'shuttleBus',
 ];
 export const defaultPaymentTypes: string[] = ['vipps', 'visa', 'mastercard'];
+export const defaultVatPercent: number = 12;
```
Comment (gorandalum) [inline src/reference-data/defaults.ts:20]: The vat percent, payment types and modes we sell tickets for are not actually reference data, and the defaults does not belong here. Rather move them closer to the `FirestoreConfigurationContext`, either in that file or in a `defaults.ts` in the same folder.

**Review by gorandalum — CHANGES_REQUESTED** [review]
Some minor changes. The changes is not directly related to your PR, as you were just following the pattern already implemented, but still would be nice to fix now.

File: src/reference-data/defaults.ts (hunk 2 above)
Comment (jorelosorio) [reply src/reference-data/defaults.ts:20]: Fixed, moved to a separated file in the same folder than `FirestoreConfigurationContext `

**Comment by jorelosorio** [comment]
@gorandalum Changes were made as requested

**Review by gorandalum — APPROVED** [review]

**Pull request merged**